src/pg/sql/42_observatory_exploration.sql

```diff
@@ -0,0 +1,116 @@
+-- Functions used to search the observatory for information
+--------------------------------------------------------------------------------
+CREATE OR REPLACE FUNCTION OBS_SEARCH(
+  search_term text
+)
+RETURNS TABLE(description text, name text, aggregate text, source text)  as $$
+BEGIN
+  RETURN QUERY
+  EXECUTE format($string$
+              SELECT description,
+                name,
+                  aggregate,
+                  replace(split_part(id,'".', 1),'"', '') source
+                  FROM observatory.OBS_column
+                  where name ilike '%%%L%%'
+                  or description ilike '%%%L%%'
```

Comment:
but that is just against a metadata table of all the columns correct? so we are talking about a table with 100s (and later small 1000s) of rows correct?